tests/service/services.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
Please use "#!/bin/bash -eu" instead. Then no entry in the ignore file will be needed.